src/parsers/js/transformers/babel6/index.js

```diff
@@ -26,7 +26,7 @@ export default {
     );
 
     return babel.transform(code, {
-      presets,
+      presets: [],
```

Comment:
> Interesting.. I kept wondering why I couldn't use any stage-0 features in the code and got unexpected token.

So, does everything work now or not? Are you saying one cannot use stage 0 features in the transform?